src/modules/coreHttpApi/controllers/AccountController.ts

```diff
@@ -23,7 +24,8 @@ export class AccountController extends BaseController {
     @Accept("application/json")
     public async sync(): Promise<Envelope> {
         const result = await this.transportServices.account.syncEverything();
-        return this.ok(result);
+        if (result.isError) return this.ok(result);
+        return this.ok(Result.ok("Sync successful"));
```

Comment:
I don't know if this was discussed or not, but IMO `"Sync successful"` is not a good return value.. wtb just making it a 204?

Reply:
I just followed the Confluence page, I agree with 204